blog/2024-03-13-ledger.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Ledger Team Update
+slug: 2024-03-13-ledger
+authors: lehins
+tags: [ledger]
+hide_table_of_contents: false
+---
+
+## High level summary
+
+Focus has been on testing: adding tests and improving our testing framework to make it easier to write tests for various Conway features. 
+We also did some cleanup and reorganisation of code, to make it more manageable and easier to release. 
+We have started work on Hardfork Initiation: support for intra-era hardfork initiation and necessary updates the SPO stake distribution calculation. 
```

Comment:
These should probably be bullet points, or separate paragraphs. In the [rendered version](https://github.com/IntersectMBO/cardano-updates/blob/bed083b8505ed28b86ceee0fcc44d9f73c95cb6d/blog/2024-03-13-ledger.md), the points are flowed together in a paragraph.

Reply:
Maybe not separate paragraphs. I see in the README for this repo:

> Each post should also start with a `## High level summary` section that will be used by technical writers to share information with the general public. The better suited this section is for the technical writers, the less chance there is for misinformation. It should ideally be one short paragraph.

Reply:
Mhm, yes good point and reminder, maybe it's better I remove the middle sentence. Thanks!